sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/_utils.py

```diff
@@ -274,17 +274,25 @@ def _filter_custom_properties(properties: Attributes, filter=None) -> Dict[str,
     return truncated_properties
 
 
-def _get_auth_policy(credential, default_auth_policy):
+def _get_auth_policy(credential, default_auth_policy, aad_audience=None):
     if credential:
         if hasattr(credential, "get_token"):
             return BearerTokenCredentialPolicy(
                 credential,
-                _APPLICATION_INSIGHTS_RESOURCE_SCOPE,
+                _get_scope(aad_audience),
             )
         raise ValueError("Must pass in valid TokenCredential.")
     return default_auth_policy
 
 
+def _get_scope(aad_audience=None):
+    # The AUDIENCE is a url that identifies Azure Monitor in a specific cloud
+    # (For example: "https://monitor.azure.com/").
+    # The SCOPE is the audience + the permission
+    # (For example: "https://monitor.azure.com//.default").
+    return _DEFAULT_AAD_SCOPE if not aad_audience else "{}/.default".format(aad_audience)
```

Comment:
Kind of hard to track, but wouldn't it already have /.default?

Reply:
Also could we reverse this just for clearer logic?

Reply:
I've left the details in the comments above this code. Audience is only the url. Scope is audience + permission so audience would not include /.default.

Reply:
Thanks